spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/builder/Imports.java

```diff
@@ -16,6 +16,17 @@
 
 package org.springframework.cloud.contract.verifier.builder;
 
-interface Imports extends Visitor<Imports> {
+import java.util.List;
+import java.util.function.Function;
+
+interface Imports extends Acceptor, Function<ClassBuilder, ClassBuilder> {
+
+	List<String> fqns();
```

Comment:
Please change to a more meaningful method name and avoid abbreviations whenever possible.

Reply:
Please add Javadoc.